client/src/components/General/Signup/Component.jsx

```diff
@@ -0,0 +1,185 @@
+        <FullPageFilter
+          onClick={() => {
+            this.props.closeSignupPanel();
+          }}
+        />
+        {this.state.signup ? (
+          <LoginContainer>
+            <LoginTitleWrapper>
+              <Logo />
+              <Brand>Harvesthru</Brand>
+            </LoginTitleWrapper>
+            <Form
+              onSubmit={this.onSubmit}
+              render={({ handleSubmit }) => (
+                <form onSubmit={handleSubmit}>
+                  <Margin top={30} bottom={15}>
+                    <Row justify="center">
+                      <Col xs={12} lg={12}>
+                        <Field
+                          name="email"
+                          type="text"
+                          validate={composeValidators(emailFormat, emailRequired)}
+                          render={({ input, meta }) => (
+                            <React.Fragment>
+                              <TextField
+                                label="Email Address"
+                                placeholder="You email address"
+                                autoFocus={false}
+                                {...input}
+                                {...meta}
+                              />
+                              {meta.touched && <FieldError>{meta.error}</FieldError>}
+                            </React.Fragment>
+                          )}
+                        />
+                      </Col>
+                    </Row>
+                  </Margin>
+                  <Margin top={15} bottom={15}>
+                    <Row justify="center">
+                      <Col xs={12} lg={12}>
+                        <Field
+                          name="password"
+                          type="password"
+                          // Note: the order of validators matter
+                          validate={composeValidators(
+                            passwordRequired,
+                            passwordLength,
+                            passwordFormat
+                          )}
+                          render={({ input, meta }) => (
+                            <React.Fragment>
+                              <TextField
+                                label="Password"
+                                placeholder="You password"
+                                autoFocus={false}
+                                {...input}
+                                {...meta}
+                              />
+                              {meta.touched && <FieldError>{meta.error}</FieldError>}
+                            </React.Fragment>
+                          )}
+                        />
+                      </Col>
+                    </Row>
+                  </Margin>
+                  <Margin top={25} bottom={15}>
+                    <Row justify="center">
+                      <Col xs={12} lg={12}>
+                        <Field
+                          name="password"
```

Comment:
NVM :) Just changed name field to `"confirmPassword"` , and it should fix the issue

Reply:
Cool thanks